adapters/rubicon/rubicon_test.go

```diff
@@ -698,15 +698,44 @@ func TestDifferentRequest(t *testing.T) {
 	// test invalid size
 	pbReq.Bidders[0].AdUnits[0].Sizes = []openrtb.Format{
 		{
-			W: 0,
-			H: 0,
+			W: 2222,
+			H: 333,
 		},
 	}
+	pbReq.Bidders[0].AdUnits[1].Sizes = []openrtb.Format{
+		{
+			W: 222,
+			H: 3333,
+		},
+		{
+			W: 350,
+			H: 270,
+		},
+	}
+	pbReq.Bidders[0].AdUnits = pbReq.Bidders[0].AdUnits[:len(pbReq.Bidders[0].AdUnits)-1]
 	_, err = an.Call(ctx, pbReq, pbReq.Bidders[0])
 	if err == nil {
-		t.Fatalf("Should have gotten an error: %v", err)
+		t.Fatalf("Should have reported an error")
 	}
 
+	pbReq.Bidders[0].AdUnits[1].Sizes = []openrtb.Format{
+		{
+			W: 222,
+			H: 3333,
+		},
+		{
+			W: 300,
+			H: 600,
+		},
+		{
+			W: 300,
+			H: 250,
+		},
+	}
+	_, err = an.Call(ctx, pbReq, pbReq.Bidders[0])
```

Comment:
You may also want to replace line 735 with this:

        bids, err = an.Call(ctx, pbReq, pbReq.Bidders[0])
	if len(bids) != 1 {
		t.Fatalf("Should have length as 1 instead of %v", bids)
	}